client/scss/components/_messages.status.scss

```diff
@@ -3,6 +3,10 @@
         color: $color-green-dark;
     }
 
+    &.duplicate {
+        color: $color-orange;
```

Comment:
I think "duplicate" is too specific for a generic component here. Try `warning`, which mirrors what we use in the messages framework?